### DISCUSSION TITLE
Clarification with Quiz_05

### DISCUSSION THREAD

**Discussion by NickTTG** [cooperrc/advanced-dynamics#41] [General]
Is Ω the same here as ω for the problem? The equations I have written for the kinetic and potential energies involve ω recalling the lecture notes so I wasn't sure if I did something wrong.

**Comment by cooperrc** [comment]
In the quiz, $\Omega$ refers to rotation of the system. So if the $\omega$ in the notes is also a rotation rate, then they would be equivalent. 

Many times, the lower-case $\omega$ is used to describe _natural frequency_ though.
